kolibri/plugins/coach/assets/src/views/plan/LessonResourceSelectionPage/LessonContentCard/index.vue

```diff
@@ -91,25 +85,8 @@
       };
     },
     props: {
-      title: {
-        type: String,
-        required: true,
-      },
-      description: {
-        type: String,
-        required: true,
-      },
-      thumbnail: {
-        type: String,
-        default: null,
-      },
-      kind: {
-        type: String,
-        required: true,
-        validator: validateContentNodeKind,
-      },
-      isLeaf: {
-        type: Boolean,
+      content: {
```

Comment:
As a tech debt cleanup, we should define a content node spec using the core validator stuff, so that we can just reuse that every time we want content/contentnode as a prop. I would be happy to be assigned to this.